### ISSUE TITLE
NSTA conference list

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/tasks#925]
**Booth 1308**

- [x] app fliers 
- [x]  general fliers 
- [x]  PhET general business cards 
- [x]  Business card holder/display
- [x]  iPad Air with app loaded
- [ ] Chromebook with app loaded
- [x] Projector
- [x] Banner (booth and popup)
- [ ] Updated video loaded on projector computer

**Comment by phet-steele** [comment]
@ariel-phet please review the video at https://github.com/phetsims/phet-info/tree/master/conferences and let me know if it should be updated.

**Comment by phet-steele** [comment]
Nevermind, @arouinfar can you review instead please.

**Comment by arouinfar** [comment]
@phet-steele I reviewed the ISTE 2017 and NSTA Stem Forum 2016 videos. I think the older NSTA video is a good one to build from. It's about a minute shorter than the ISTE video, which is why I've recommended more adding than swapping/removing.

- [ ] Add CCK: DC
- [ ] Add Projectile Motion
- [ ] Add Gravity and Orbits
- [ ] Add States of Matter (use ISTE clip)
- [ ] Remove Graphing Lines
- [ ] Swap Neuron with Gene Expression Essentials (optional)
